### DISCUSSION TITLE
Extract command not working correctly

### DISCUSSION THREAD

**Discussion by dyoung-ess** [dbeaver/dbeaver#16462] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10 Pro version 10.0.19043 build 19403
- DBeaver version: Version 22.0.4.202205011916
- Additional extensions: Using JDBC driver

#### Connection specification:
- Database name and version: Bigquery
- Driver name: com.simba.googlebigquery.jdbc42.Driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
SQL: extract(year from date) as year is extracting the wrong year when using a date value of [2022-01-31 18:00:00.000]

**Comment by ShadelessFox** [comment]
Hello @dyoung-ess,

I'm unable to reproduce your issue:

![image](https://user-images.githubusercontent.com/35821147/167945179-f7533246-c8b0-44da-a402-92e88167b491.png)

Can you please provide more information, such as the actual output?

**Comment by dyoung-ess** [comment, marked as answer]
I found my answer to the issue.  Since I'm using Bigquery it defaults time
to UTC and the JVM in the JDBC driver wasn't.  I had to set the driver
properties for TimezoneFallback to 1 instead of blank or 0.
